modules/homestakeros/options.nix

```diff
@@ -320,6 +320,28 @@
       };
 
       addons = {
+        ssv-node = {
+          privateKeyFile = mkOption {
+            type = types.nullOr types.path;
+            default = "/var/mnt/secrets/ssv/ssv_operator_key";
+            description = "Path to the private SSV operator key.";
+          };
+          execEndpoint = mkOption {
+            type = types.str;
+            default = "ws://127.0.0.1:8551";
+            description = "WebSocket URL of the execution node to connect to.";
+          };
+          consEndpoint = mkOption {
```

Comment:
cons might be confusing name: https://en.wikipedia.org/wiki/Cons

we should it name a consensus endpoint like to keep consistent naming scheme, or better yet, use the already defined variables to preselect it (at least as the default value). i personally think we do not necessarily need to provide an option on this level.

Reply:
Let's rework the RPC daemon first and then we can make both of the options dynamic.